msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/SignInErrors.kt

```diff
@@ -62,3 +62,22 @@ class SignInSubmitPasswordError(
 ): SignInSubmitPasswordResult, Error(errorType = errorType, error = error, errorMessage= errorMessage, correlationId = correlationId, errorCodes = errorCodes, exception = exception) {
     fun isInvalidCredentials(): Boolean = this.errorType == SignInErrorTypes.INVALID_CREDENTIALS
 }
+
+/**
+ * Sign in continuation error. The error has no utility methods
+ * and must be treated as unexpected. This error is produced by
+ * [com.microsoft.identity.nativeauth.INativeAuthPublicClientApplication.signIn]
+ * @param errorType the error type value of the error that occurred
+ * @param error the error returned by the authentication server.
+ * @param errorMessage the error message returned by the authentication server.
+ * @param correlationId a unique identifier for the request that can help in diagnostics.
+ * @param errorCodes a list of specific error codes returned by the authentication server.
+ * @param exception an internal unexpected exception that happened.
+ */
+open class SignInContinuationError(
+    override val error: String? = null,
+    override val errorMessage: String?,
+    override val correlationId: String,
```

Comment:
can you put a TODO that correlationId should be `String?` and needs to be changed after Sammy's PR is merged.